src/ui/utils/telemetry.ts

```diff
@@ -34,11 +34,20 @@ export function setupTelemetry(context: Record<string, any>) {
   Sentry.setContext("recording", { ...context, url: window.location.href });
 }
 
+type TelemetryUser = {
+  userId: string | undefined;
+  userEmail: string | undefined;
+  isInternal: boolean;
+};
+
+let telemetryUser: TelemetryUser;
+
 export function setTelemetryContext(
   userId: string | undefined,
   userEmail: string | undefined,
   isInternal: boolean
 ) {
```

Comment:
should we refactor this function to also receive the TelemetryUser? `export function setTelemetryContext({user: TelemetryUser})`

it's probably only called from one spot so no big deal